cmd/payload-testing-prow-plugin/server.go

```diff
@@ -44,13 +46,32 @@ func helpProvider(_ []prowconfig.OrgRepo) (*pluginhelp.PluginHelp, error) {
 	return pluginHelp, nil
 }
 
+type trustedChecker interface {
+	trustedPullRequest(author, org, repo string, num int) (bool, error)
+}
+
+type githubTrustedChecker struct {
+	githubClient github.Client
+	pluginAgent  *plugins.ConfigAgent
+}
+
+func (c *githubTrustedChecker) trustedPullRequest(author, org, repo string, num int) (bool, error) {
+	_, trusted, err := trigger.TrustedPullRequest(
```

Comment:
Hmm... Why this and not `TrudstedUser`:

https://github.com/kubernetes/test-infra/blob/ee1e7c845bb25177944bb5de37545f007f529059/prow/plugins/trigger/trigger.go#L221-L223

IMO we just need to validate who does the `/payload`, we do not care about the actual PR.

Reply:
Updated.